client/src/components/forms/MailingList/slides/Social.tsx

```diff
@@ -23,54 +24,131 @@ const styles = (theme: Theme) =>
       flexDirection: 'column',
       marginBottom: theme.spacing(6),
     },
+    centerText: {
+      textAlign: 'center',
+    },
     iconsContainer: {
       display: 'flex',
       justifyContent: 'space-around',
       width: '100%',
+      flexWrap: 'wrap',
+      marginBottom: theme.spacing(2),
+      [theme.breakpoints.down('sm')]: {
+        marginBottom: 0,
+      },
     },
     icon: {
       width: ICON_SIZE,
       height: ICON_SIZE,
     },
+    button: {
+      [theme.breakpoints.down('sm')]: {
+        marginBottom: theme.spacing(1),
+      },
+    },
+    scContainer: {
+      cursor: 'pointer',
+    },
+    sc: {
+      maxWidth: '100%',
+      maxHeight: '100%',
+    },
+    snapCodeWrapper: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    snapCodeContainer: {
+      height: 400,
+      [theme.breakpoints.down('sm')]: {
+        height: 'unset',
```

Comment:
Since this is a square image I think we should be fine only setting and overriding widths and that way we don't have to unset anything.

Reply:
Ah yea we don't need that one